onboarding.qmd

```diff
@@ -138,6 +77,6 @@ If you haven't already, please create a professional GitHub account (e.g., `jana
 
 # Social media
 
-If you have officially joined us as a grad or postdoc, or undergrad for 3/6+ months, you will automatically get added to our [group webpage](https://jravilab.github.io/#people). If not, please check a few sample pages and furnish me with those details -- I can add you there right away. Also, we use Twitter for professional networking & announcements (connecting with the broader scientific/R/technical communities). If you have a Twitter handle, share it with me (DM), or follow #auto-twitter for updates. You can check out/follow a few accounts maintained by us: [JRaviLab](https://twitter.com/jravilab) \| [my handle](https://twitter.com/janani137) \| [RLadies-Aurora](https://twitter.com/rladies-aurora) \| [RLadies-EastLansing](https://twitter.com/rladies-eastlansing) \| [Women+ Data Science](https://twitter.com/womenplusdata) \| [AsiaR](https://twitter.com/AsiaR-comm) \| [ISMB EvolCompGen COSI](https://twitter.com/EvolComp). Thanks to recent *interesting* changes at Twitter, people have been migrating to Mastodon (e.g., [genomic.social](https://genomic.social/explore), [fosstodon](https://fosstodon.org/explore))!
+If you have officially joined us as a grad or postdoc, or undergrad for 3/6+ months, you will automatically get added to our [group webpage](https://jravilab.github.io/#people). If not, please check a few sample pages and furnish me with those details -- I can add you there right away. Also, we use Twitter for professional networking & announcements (connecting with the broader scientific/R/technical communities). If you have a Twitter handle, share it with me (DM), or follow #auto-twitter for updates. You can check out/follow a few accounts maintained by us: [JRaviLab](https://twitter.com/jravilab) \| [my handle](https://twitter.com/janani137) \| [RLadies-Aurora](https://twitter.com/rladies-aurora) \| [RLadies-EastLansing](https://twitter.com/rladies-eastlansing) \| [Women+ Data Science](https://twitter.com/womenplusdata) \| [AsiaR](https://twitter.com/AsiaR-comm) \| [ISMB EvolCompGen COSI](https://twitter.com/EvolComp). Thanks to recent *interesting* changes at Twitter, people have been migrating to Mastodon (e.g., [genomic.social](https://genomic.social/explore), [fosstodon](https://fosstodon.org/explore)) and Bluesky \| [my handle](https://bsky.app/profile/jananiravi.bsky.social)!
```

Comment:
Did anything change?

Reply:
This is the one where there is a change, just adding Bluesky. I'm not sure how this became outdated and the duplicate change of seemingly nothing popped up as a second edit.